tests/PactNet.Native.Tests/Verifier/NativePactVerifierConsumerTests.cs

```diff
@@ -0,0 +1,56 @@
+using System.IO;
+using System;
+using Xunit;
+using System.Collections.Generic;
+using FluentAssertions;
+using PactNet.Native.Verifier;
+using PactNet.Verifier;
+using Xunit.Abstractions;
+
+namespace PactNet.Native.Tests.Verifier
+{
+    public class NativePactVerifierConsumerTests
```

Comment:
I would add edge cases like empty space, null arguments etc as combability tests.

For instance

[Theory]
[InlineData(null)]
[InlineData("")]
[InlineData(" ")]
public void BasicAuthentication_Fails_WhenCalled_WithInvalidArguments(string authArg)
{
//I verify here an exception is thrown for instance.
}